features/search.feature

```diff
@@ -1,13 +1,36 @@
-Feature: Searching
+Feature: Searching on MAS site
   As a user visiting the website
   I want to search by keyword
   So that I can find the content I am looking for
 
-  Scenario: The search box is visible in the header
-    When I visit the home page
-    Then I should see the search box
-
-  Scenario: Search can be performed
+  Background:
     Given I am on the home page
-    When I search for something relevant
-    Then I should see search results
+
+  Scenario: Search on first result page
+    When I search the query "money"
+    Then I should see "10" results
+    And I should be on page "1" of "36" of the search results
+    And I should not see the previous button
+
+  Scenario: Search and navigate to the next result page
+    When I search the query "money"
+    And I go to the next results page
+    Then I should see "10" results
+    And I should be on page "2" of "36" of the search results
+
+  Scenario: Search and navigate to last result page
+    When I search the query "money"
+    And I go to results page "35" of the "money" search query
```

Comment:
I'd still wonder whether to reword this step. I think it could confuse the business logic of the feature to take into account zero indexing. It isn't currently a reusable step by the look of it and so could easily be reworded. wdyt?

Reply:
Thank you for the clarification. But the "35" would be hardcoded on the step. Then it could trick who uses the step *in case the last page is not 35*. Am I missing something?

Reply:
ok, I see your point. I think because it currently isn't being reused it would be ok to be hardcoded. If we want to reuse it in the future we can refactor? we could always have a comment as to why it is 35 as well?

Reply:
♻️